src/Adapters.Tests/Log4NetAppender.Tests.Shared/ApplicationInsightsAppenderTests.cs

```diff
@@ -1,11 +1,4 @@
-﻿// -----------------------------------------------------------------------
-// <copyright file="ApplicationInsightsAppenderTests.cs" company="Microsoft">
-// Copyright (c) Microsoft Corporation. 
-// All rights reserved.  2013
-// </copyright>
-// -----------------------------------------------------------------------
```

Comment:
It is not complaining about this one and in Core and Web we removed all headers and the rule. But I can put it back though I think this comment is useless.

---
In reply to: [66371291](https://github.com/Microsoft/ApplicationInsights-dotnet-logging/pull/34#discussion_r66371291) [](ancestors = 66371291)